api/notifications_and_subscriptions.rb

```diff
@@ -19,9 +19,9 @@
 end
 
 post "#{APIPREFIX}/users/:user_id/subscriptions" do |user_id|
-  user.subscribe(source).to_hash.to_json
+  user.subscribe(source)
```

Comment:
Can you jot a note about this change?  Do you know why this was included if it wasn't needed?